service/src/main/resources/db/migration/V28__create_finance_decision_manager_field.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE daycare ADD COLUMN finance_decision_manager uuid;
+ALTER TABLE daycare ADD FOREIGN KEY (finance_decision_manager)
+REFERENCES employee(id) ON DELETE SET null;
```

Comment:
I don't know if you did this in separate steps on purpose, but you can combine this to a single statement `ALTER TABLE daycare ADD COLUMN finance_decision_manager uuid REFERENCES employee(id) ON DELETE SET NULL;`